msg/AirspeedWind.msg

```diff
@@ -9,6 +9,7 @@ float32 variance_east		# Wind estimate error variance in east / Y direction (m/s
 
 float32 tas_innov 		# True airspeed innovation
 float32 tas_innov_var 		# True airspeed innovation variance
+float32 tas_innov_integ_test_ratio  # Value > 1 indicates that innovation failure check has triggered
```

Comment:
It's not really a test ratio, is it? Should it be called "airspeed_innov_exceeds_threshold?

Overall though, are we sure that this adds value? In the end, the innovation is already logged and the threshold is just a parameter, so it can already be quite easily deferred from the logged data atm.

Reply:
That being said, I don't think logging this field is absolutely required. I can definitely understand the point of not increasing the topic sizes and can see that this is not a field that you will use a lot.

Reply:
Good point about the integration, that makes it harder to spot.